app/views/customers/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1 id='customer_name'>Customer Name: <%= @customer.name %></h1>
+
+
+<ul id="item_list">
+  <% @customer_items.each do |item| %>
+    <li><%= "#{item.name}, $#{item.price}, Supermarket: #{item.supermarket.name}" %></li>
+  <% end %> 
+</ul>
+
+
+<div id='add_item'>
+  <%= form_with url: "/customers/#{@customer.id}", method: :get, data: { turbo: false } do |form| %>
```

Comment:
So this form is to add an item to a customer, so what we are actually wanting to do here is use that join association between customer and item (CustomerItem). So we would want our path to be
```ruby
"/customers/#{customer.id}/items", method: :post 
```
So this would then require us to create a route of the same nature but then we could point it to `customer_items_controller` with a `create` action.